added upper-case-solution (#8270)
github/docs: data/reusables/pages/create-repo-name.md

> 1. Type a name for your repository and an optional description. If you're creating a user or organization site, your repository must be named `<user>.github.io` or `<organization>.github.io`. If `<user>` or `<organization>` contains uppercase letters, you need to lowercase them.

Reviewer: We avoid pronouns whenever possible, as they can be ambiguous.

```suggestion
1. Type a name for your repository and an optional description. If you're creating a user or organization site, your repository must be named `<user>.github.io` or `<organization>.github.io`. If `<user>` or `<organization>` contains uppercase letters, you must lowercase the letters.
```
